fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/api/rdf/HttpResourceConverter.java

```diff
@@ -449,4 +449,9 @@ private String txSegment() {
             return EMPTY;
         }
     }
+
+    @Override
```

Comment:
Yes, it is not clear to me that overriding `hashCode()` is something we want to do.
Additionally, I am not seeing an indication in Sonar that would suggest we override `hashCode()`.